### ISSUE TITLE
Cookie set in initial page load is not passed to fetch in load function 

### ISSUE THREAD

**Issue by pbvahlst** [sveltejs/kit#5210]
### Describe the bug

When a page is using `fetch` in it's `load` function the `fetch()` call does not include the cookie returned by the initial call to the page. 

### Reproduction

https://stackblitz.com/edit/sveltejs-kit-template-default-ze4xq5?file=src%2Froutes%2Findex.svelte

notice how `NO COOKIE` is printed twice when loading the page for the first time. 

### System Info

```shell
See stackblitz
```

### Severity

annoyance

**Comment by tcc-sejohnson** [comment]
I don't know whether this is intended behavior or not, but it seems like at least _logical_ behavior. When you say "...the `fetch()` call does not include the cookie returned by the initial call to the page," I believe you're juuuust barely off of what's actually happening. In reality, the `fetch` in `load` is being run _as part of_ the initial call to the page, i.e. _before_ the cookie is returned from `handle` to the client. It's running on the server _during_ `await resolve(event)`, but prior to `return response`. I added some logging statements to your example so you can see what's going on: https://stackblitz.com/edit/sveltejs-kit-template-default-26iwxj?file=src/hooks.js

In order for `load`'s `fetch` to send a cookie you added in `handle` back to the server, it would have to be called as part of a separate navigation, because the response from `handle` would have to be returned to the client in order to set the cookie on the browser. It would _also_ have to be called on the client, as it would not have access to your client-side cookies when running on the server during SSR.

TL;DR, if you need to access data in `load` from the `handle` pipeline, I'd recommend doing so through `session`, as it's never guaranteed that the `fetch` in `load` (which can run on the server) is going to have access to the browser's cookies.

**Comment by Conduitry** [comment]
Yes, @tcc-sejohnson is correct. Because of SSR, the `fetch` in `load` is being called as part of the initial request. The first request hasn't even finished yet, so there wouldn't even be a way to access that cookie. If you add a `console.log('fetching');` before the `fetch` in your `load` function, you'll see that it happens before both instances of `NO COOKIE`.

Closing as working as intended.

**Comment by pbvahlst** [comment]
Thanks, I figured that it was what was going on, it just makes i problematic to bind a session object to a session cookie on the initial call, when `load` is used. But I'll work arround it.
